app/code/Magento/Translation/Model/FileManager.php

```diff
@@ -73,7 +73,7 @@ public function getTranslationFileTimestamp()
     /**
      * @return string
      */
-    protected function getTranslationFileFullPath()
+    public function getTranslationFileFullPath()
```

Comment:
Change of method visibility may break potential customization or extension which inherited from this class.